app/src/main/res/values/strings.xml

```diff
@@ -86,4 +86,10 @@
     <string name="title_select_date">Select date</string>
     <string name="title_select_start_day">Start date</string>
     <string name="title_select_end_day">End date</string>
+    <string name="title_skip_login">Skip Login</string>
+    <string-array name="title_status">
```

Comment:
WARNING: [UnusedResources] The resource `R.array.title_status` appears to be unused
 Unused resources make applications larger and slow down builds.